tests/integration/test_run_simulation.py

```diff
@@ -98,13 +98,26 @@ def test_run_simulation_with_flask(celery_app, celery_worker, client: Flask, db_
         sleep(0.5)
     assert i < max_number_of_attempts - 1, "Job did not finish in time"
 
-    logging.info("Fetching results from /results endpoint")
+    # Test the results endpoint without estimator_name
+    logging.info("Fetching results from /results endpoint without estimator_name")
     resp = client.get("/results", query_string={"job_id": job_id})
     results_data: dict = json.loads(resp.data.decode())
 
     assert resp.status_code == 200  # skipcq: BAN-B101
     assert {"message", "estimators"} == set(results_data.keys())
 
+    # Test the results endpoint with a specific estimator_name
```

Comment:
Can you test as well what will happen if non-existing estimator_name will be passed ?